docs/pages/docs/guide/advanced/table.mdx

```diff
@@ -171,19 +173,29 @@ Just wrap your table with curly braces `{` and `}`, e.g.
 </table>}
 ```
 
-#### Changing Default Behaviour
+Alternatively, create a reusable table component:
+
+```mdx filename="MDX"
+export function Table({ columns, data }) {
+  return <table>{/* ... */}</table>
+}
+
+<Table columns={...} data={...} />
+```
+
+### Changing Default Behaviour
 
 If this thing is still confusing for you, and you want to use regular literal
 HTML elements for your tables, do the following:
 
 <Steps>
-### Install `remark-mdx-disable-explicit-jsx` package
+#### Install `remark-mdx-disable-explicit-jsx` package
```

Comment:
In Nextra to style `<details>` and `<summary>` we don't use this package but we use a custom remark plugin, maybe we should introduce a new option in `nextraConfig.mdxOptions` and pass white list tags to below ?

https://github.com/shuding/nextra/blob/af8c38be432ed2bc719c378af4a0f6c650389716/packages/nextra/src/server/compile.ts#L231-L236

Reply:
Thanks for pointing that out! I didn’t realize Nextra has this remark plugin. It seems more convenient than the current approach.

Reply:
You are welcome!